corehq/apps/reports/static/reports/js/case_activity.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+hqDefine("reports/js/case_activity", function () {
+
+    $(document).ajaxSuccess(function (event, xhr, settings) {
+        if (settings.url.match(/reports\/async\/case_activity/)) {
+            const $specialNotice = $("#report-special-notice");
+            if ($specialNotice) {
+                const urlParams = new URLSearchParams(window.location.search);
+                (urlParams.get('view_by') === 'groups') ? $specialNotice.show() : $specialNotice.hide();
```

Comment:
Does this make it possible to show/hide the banner instantly when user selects a certain option in the UI? or its just at the time of page load?
if its latter, might be better to just add/skip the section in the template.

Reply:
Okay, just checking.
How about we keep it simple and just show that message as a help text on that option itself next to "View by users/groups"? in a information icon? and not as a banner that we show/hide.

Reply:
This filter is also being used in the Worker Activity report [here](https://github.com/dimagi/commcare-hq/blob/0af7ed0eb24093fe68c04141dccf616c0e51c1ef/corehq/apps/reports/standard/monitoring.py#L1279). The Worker Activity report data includes cases which aren't directly owned by a specific group, and so the help text wouldn't hold true in this scenario.

Reply:
Okay.

Fine to leave it as it is @zandre-eng. Thanks for talking this through.